devices/System.Drawing/System.Drawing.cs

```diff
@@ -180,9 +180,82 @@ public float GetSaturation()
         }
 
         /// <Summary>Gets the 32-bit ARGB value of this System.Drawing.Color structure.</Summary>
-        /// <returns>The 32-bit ARGB value of this System.Drawing.Color.</returns>     
+        /// <returns>The 32-bit ARGB value of this System.Drawing.Color.</returns>
         public int ToArgb() => (int)_color;
 
+        /// <summary>
+        /// Internal Helper function for ParseHex
+        /// </summary>
+        private static int ParseHexChar(char c)
+        {
+            const int zeroChar = (int)'0';
+            const int aLowerChar = (int)'a';
+            const int aUpperChar = (int)'A';
+            var intChar = (int)c;
```

Comment:
why do you use an int and cast to int? char comparison should just work as fine.